CHANGELOG.next.md

```diff
@@ -61,6 +61,8 @@ Thanks, you're awesome :-) -->
 
 #### Added
 
+ * Added RFC for session fields #879
```

Comment:
Perhaps holding off on `CHANGELOG` entries for RFCs would be best until field definitions adds/changes would be committed later (stage 2)?

Any thoughts @webmat @epixa?

Reply:
Discussed with @webmat - agreed the CHANGELOG entry isn't necessary until a later stage.